frontend/src/metabase/search/components/SearchFilterModal/SearchFilterModal.tsx

```diff
@@ -0,0 +1,89 @@
+import { t } from "ttag";
+import { useEffect, useMemo, useState } from "react";
+import Modal from "metabase/components/Modal";
+import { SearchFilterModalFooter } from "metabase/search/components/SearchFilterModal/SearchFilterModalFooter";
+import {
+  FilterTypeKeys,
+  SearchFilterComponent,
+  SearchFilterKeys,
+  SearchFilters,
+} from "metabase/search/types";
+import Button from "metabase/core/components/Button";
+import { Title, Flex } from "metabase/ui";
+import { TypeFilter } from "./filters/TypeFilter";
+import { SearchFilterWrapper } from "./SearchFilterModal.styled";
+
+const filterMap: Record<FilterTypeKeys, SearchFilterComponent> = {
+  [SearchFilterKeys.Type]: TypeFilter,
+};
+
+export const SearchFilterModal = ({
+  isOpen,
+  setIsOpen,
+  value,
+  onChangeFilters,
+}: {
+  isOpen: boolean;
+  setIsOpen: (isOpen: boolean) => void;
+  value: SearchFilters;
+  onChangeFilters: (filters: SearchFilters) => void;
+}) => {
+  const [output, setOutput] = useState<SearchFilters>(value);
```

Comment:
I can understand the reason behind `output` but the name itself doesn't help readers understand what it is. Could/should this be renamed to something along the line of `searchFilters`/`setSearchFilters` (using the type as their name)?